application/extensions/AdvancedSettingWidget/views/integer.php

```diff
@@ -4,17 +4,15 @@
             <?= $this->setting['aFormElementOptions']['inputGroup']['prefix']; ?>
         </div>
     <?php endif; ?>
-        <!--
-        max=""
-        min=""
-        -->
     <input 
         type="number" 
         class="form-control" 
         name="<?= $inputBaseName ?>"
         id="<?= CHtml::getIdByName($inputBaseName); ?>"
         <?= ($this->setting['help']) ? 'aria-describedby="help-' . CHtml::getIdByName($inputBaseName) . '"' : "" ?>
         value="<?= CHtml::encode($this->setting['value']); ?>"
+        <?= isset($this->setting['min']) ? 'min="' . $this->setting['min'] . '"' : '' ?>
```

Comment:
Checking like this is OK or should also check there is content?

Reply:
What do you mean ?

Reply:
> if the min setting contains a space, should the attribute be used or not?

If it's not an integer :). 
It's PHP or config.xml file : it's not user entered data. I think we don't have to fix it.

`value="<?= CHtml::encode($this->setting['value']); ?>"` don't check if it's integer here too :)

Reply:
Use `??` here please.

Reply:
Ah wait, not possible. Sorry, my bad.

Reply:
In 3.X : we have a system to be sure to have something : https://github.com/LimeSurvey/LimeSurvey/blob/06e252130069f697b247446ae52c6b8b18d261d8/application/models/QuestionAttribute.php#L312

But in 5.X … no system